css/styles.css

```diff
@@ -0,0 +1,201 @@
+* {
+  font-family: Merriweather, Source Sans Pro, Oswald;
+  font-weight: lighter;
+}
+
+#container {
+  width: 875px;
+  margin: auto;
+ }
+
+header {
+  background-color: black;
+  width: 100%;
+  height: 100px;
+  border-bottom: 3px solid yellow;
+  display: flex;
+  padding-bottom: 10px;
+  float: left;
+ }
+
+ nav{
+  text-align: right;
+  display: flex;
+  align-items: flex-end;
+  justify-content: space-around;
+ } 
+
+ .logo{ 
+  color: yellow;
+  font-size: 28px;
+  padding: 15px;
+  }
+
+ .tagline {
+  background-color: lightgray;
+  width: 100%;
+  height: 10%;
+  justify-content: center;
+  float: left;
+  margin-top: 0px;
+ }
+
+ .tagline p {
+  text-align: center;
+  font-style: italic;
+  color: black;
+  vertical-align: middle;
+  margin-bottom: 10px;
+ }
+
+span-taste{
+  color: grey;
+  text-decoration: line-through;
+}
+span-are{
+  text-decoration: underline yellow;
+}
+
+ a {
+  color: white;
+  text-decoration: none;
+  font-family: impact, sans-serif;
+  padding-right: 20px;
+ }
+
+ .standout-font{
+  text-transform: uppercase;
+  font-size: 18px;
+  font-weight: bolder;
+ }
+
+ a:hover {
+  color: yellow;
+ }
+
+footer {
+  background-color: black;
+  border: 1px black solid;
+  display: flex;
+  height: 40px;
+  float: left;
+  width: 100%;
+ /* font-family: sans-serif;*/
+ }
+
+ h4 {
+  vertical-align: middle;
+  font-family: sans-serif;
+  padding-left: 10px;
+  color: white;
+ }
+
+.main {
+  background-image: url("../images/coworking.jpg");
+  background-position: center;
+  height: 83vh;
+  width: 100%;
+  float: left;
+}
+
+.main-overlay {
+  background-color: grey 
+  opacity: 0.7;
+  color: black;
+  border: 1px solid lightgray;
+  width: 250px;
+  height: 400px;
+  float: right;
+  padding: 25px;
+  margin: 25px;
+}
+
+.main-overlay p {
+  font-size: 18px;
+  font-family: sans-serif;
+}
+
+aside {
+  width: 100%;
+  height: 45vh;
+ /* float: left;*/
+  display: flex;
+}
+
+.section-container {
+  display: flex;
+  float: left;
+  width: 100%;
+  height: 100%;
+  justify-content: center;
+}
+
+aside p {
+  float: left;
+  width: 200px;
+  margin: 5px;
+  padding: 5px;
+  border-right: solid 1px lightgrey
+}
+
+aside a {
+  color: black;
+  text-transform: uppercase;
+  background-color: lightgray;
+}
+
+aside a:hover {
+  color: red;
+}
+
+article a {
+  color: black;
+  background-color: yellow;
+  text-transform: uppercase;
+  te
+}
+
+article a:hover {
+  color: darkgreen;
+}
+
+.button {
```

Comment:
@AmyMirisola since .button and .grey-button share the same style except for the color. You could declare this as ```.button, .grey-button {..styles..}``` then have `.grey-button { background: lightgrey; }`